### ISSUE TITLE
Feature Request - Support for more than 2 levels of navigation

### ISSUE THREAD

**Issue by garrettoptland** [Kademi/kademi-dev#6131]
The menu component currently only allows for 2 levels of navigation (e.g. 1 sub menu). 

Ideally the menu should be able to show up to 2 or even 3 sub menus, as all other modern cms currently have this feature

**Comment by bradmac** [comment]
Kademi's standard menu component is based on the bootstrap  navbar, which supports 2 level menus.

However, the menu editor and framework supports any level of menu nesting, so the menu editor allows you to build any level of menu structure you want. You just need to customise the menu component to display how you prefer, or even create your own component

I think it would be nice to have a menu component like this
![image](https://user-images.githubusercontent.com/826458/46858864-c3b2dd80-ce68-11e8-92df-a8c06c9df6de.png)

**Comment by bradmac** [comment]
Although usually deep menu structures like that wouldnt be managed in a single menu builder. The items would usually come from different sections of the CMS, eg from elearning, blogs, forums, etc.

@garrettoptland can you give an example of what you're after? what would be the items in the menu for your use case? ie just pages in the website?

Note that for large content collections i would suggest using the Asset Management system, rather then plain html pages.

**Comment by garrettoptland** [comment]
Yeah that first example is nice, even just something basic like these:

https://bootsnipp.com/snippets/featured/multi-level-dropdown-menu-bs3

https://www.w3schools.com/bootstrap/tryit.asp?filename=trybs_ref_js_dropdown_multilevel_css&stacked=h

Menu items are just for pages in the website. Advised client against nesting menu items too deeply but menu structure was mostly taken and tweaked from their previous website
